MAUI/Rich-Text-Editor/Toolbar.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Toolbar in .NET MAUI Rich Text Editor | Syncfusion®
+description: Learn here all about Toolbar features in Syncfusion® .NET MAUI Rich Text Editor (SfRichTextEditor) control.
+platform: maui 
+control: Rich Text Editor
+documentation: ug
+---
+
+# Toolbar in .NET MAUI Rich Text Editor (SfRichTextEditor)
+
+## Toolbar position
+
+The Rich Text Editor allows you to position the toolbar at the top or bottom of the content area, depending on your layout requirements. By default, the toolbar appears at the top on Windows and macOS, and at the bottom on Android and iOS for better accessibility.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<rte:SfRichTextEditor ToolbarPosition="Bottom" />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRichTextEditor richTextEditor = new SfRichTextEditor();
+richTextEditor.ToolbarPosition = RichTextEditorToolbarPosition.Bottom;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Link quick toolbar
```

Comment:
mention as tooltip, and add notes as it will be hide after 2 sec if no interaction performed.